### ISSUE TITLE
Sensible default for LrScheduleHandler epoch_level

### ISSUE THREAD

**Issue by matt3o** [Project-MONAI/MONAI#6625]
**Describe the bug**
The default setting for LrScheduleHandler is epoch_level which contradicts or even breaks torch functionality.
The more specific bug exists in the DeepEdit Code I got from my supervisor which is very similar to the code here: https://github.com/Project-MONAI/tutorials/blob/main/deepedit/ignite/train.py
Here the lr_scheduler has a step size of 5000 however the LrScheduleHandler calls lr_scheduler.step() only after each epoch. This means the learning rate never decreases unless the model is trained for 5000 epochs. This is of course not the intented behaviour - the authors assumed that scheduler.step() gets called after each iteration.

This of course is only a concrete bug which could be fixed. Imho however the default settings for MONAI should be epoch_level == False since this is what people would expect. The fact that the switch exists is really cool, but the default should be off such that using / converting Pytorch code to MONAI works as expected.

**Comment by KumoLiu** [comment]
Hi @matt3o, thanks for your interest here. I think the default behavior in PyTorch also assumes that lr_scheduler is based on epoch level.
https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html

I think maybe we should update the tutorial you posted.
Thanks!

**Comment by matt3o** [comment]
Oh that is absolutely correct. I was sure I checked but maybe I had some older tutorials. Thanks for checking!
Also yeah, fixing the DeepEdit learning rate should be enough then.
